app/views/steps/check/results/show.en.html.erb

```diff
@@ -41,7 +47,37 @@
         Read more about this on GOV.UK</a>.
     </p>
 
-    <h3 class="govuk-heading-m" id="disclosure-and-barring-service">Disclosure and Barring Service (DBS) checks</h3>
+    <% if @presenter.motoring? %>
+      <h3 class="govuk-heading-m">Your driving record</h3>
+
+      <p class="govuk-body">
+        This checker tells you whether cautions or convictions are spent.
+        However, some motoring endorsements stay on your driving record after the caution or conviction is spent.
+      </p>
+
+      <p class="govuk-body">
+        Your driving record is maintained by the Driver and Vehicle Licensing Agency (DVLA).
+        You can <a href="https://www.gov.uk/view-driving-licence" class="govuk-link" rel="external" target="_blank">view your driving licence information</a>, including your driving record, on GOV.UK.
+      </p>
+
+      <p class="govuk-body">
+        You can find out more about <a href="https://www.gov.uk/penalty-points-endorsements" class="govuk-link" rel="external" target="_blank">penalty points</a> on GOV.UK.
+      </p>
+    <% end %>
+
+    <% if @presenter.time_on_bail? %>
+      <h3 class="govuk-heading-m">
+        Time spent on bail with an electronic tag
+      </h3>
+
+      <p class="govuk-body">
+        Your result and spent date include the number of days you said you spent
+        on bail with an electronic tag that counted towards your sentence.
+        Each full day offsets a day from the total length of your sentence.
+      </p>
+    <% end %>
+
+    <h3 class="govuk-heading-m">Disclosure and Barring Service (DBS) checks</h3>
```

Comment:
As part of the merge the ID (needed for DBS info links) has been removed in this header:
`id="disclosure-and-barring-service"`